eng/pipelines/templates/stages/cosmos-sdk-client.yml

```diff
@@ -94,17 +94,25 @@ stages:
             DisplayName: 'Emulator only Integration Tests'
             PROTOCOLS: '["Tcp"]'
             DESIRED_CONSISTENCIES: '["Strong", "Session"]'
+            AdditionalArgs: '-DargLine="-DACCOUNT_HOST=https://localhost:8081/"'
           Https_Integration_Tests_Java8:
             OSVmImage: 'windows-2019'
             JavaTestVersion: '1.8'
             ProfileFlag: '-Pemulator'
             DisplayName: 'Emulator only Integration Tests'
             PROTOCOLS: '["Https"]'
             DESIRED_CONSISTENCIES: '["Strong", "Session"]'
+            AdditionalArgs: '-DargLine="-DACCOUNT_HOST=https://localhost:8081/"'
+          Spring_Integration_Tests_Java8:
+            OSVmImage: 'windows-2019'
+            JavaTestVersion: '1.8'
+            ProfileFlag: '-P integration-test-emulator'
+            DisplayName: 'Spring Emulator only Integration Tests'
+            AdditionalArgs: '-DargLine="-DACCOUNT_HOST=https://localhost:8081/ -DACCOUNT_KEY=C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw== -DSECONDARY_ACCOUNT_KEY=C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw=="'
```

Comment:
Are there any security concerns here? Is this secret used anywhere else, can it be randomly generated?

Reply:
No security concerns : this key is documented in public Microsoft docs : https://docs.microsoft.com/en-us/azure/cosmos-db/local-emulator#authenticating-requests
Also in azure-cosmos source code: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/TestConfigurations.java#L42